tests/mock_smtp.py

```diff
@@ -70,7 +70,7 @@ def loop(self):
         exit.
         """
         while len(asyncore.socket_map):
-            asyncore.loop(timeout=1, use_poll=True)
+            asyncore.loop(timeout=0, use_poll=True)
```

Comment:
Was there a reason for this change?  When I last touched this file it was to prevent it from using 100% cpu because it was in a tight loop.  Does this put it back into that state?